exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchClient.java

```diff
@@ -162,7 +162,8 @@ private void exportBulk() {
 
     final BulkIndexResponse response;
     try {
-      response = MAPPER.readValue(httpResponse.getEntity().getContent(), BulkIndexResponse.class);
+      final var responseBody = httpResponse.getEntity().getContent();
+      response = MAPPER.readValue(responseBody.readAllBytes(), BulkIndexResponse.class);
```

Comment:
:thinking: I'm wondering why we read the response in all theses cases even if the response might be failed (4x, 5x)?

Reply:
The client will throw an exception on 4xx or 5xx, so there's no need to worry about it here. What happened in the support case is the response was not an error, but it was a proxy returning some content.